openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/widgets/SpacerWidget.vue

```diff
@@ -21,7 +21,7 @@
 -->
 
 <template>
-  <div :style="style" />
+  <div :style="localStyle" />
```

Comment:
Any idea why a simple name change fixed this?

Reply:
Style is defined in the Widget.js mixin.   Please evaluate and make sure that is working like expected.